### PR TITLE
Update Envoy SHA to latest with deterministic hash (master).

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Sigh, we need Bazel 0.22 in build images.

File: WORKSPACE [hunk 1]
```diff
@@ -35,21 +35,25 @@ bind(
...
 
-ENVOY_SHA256 = "79629284ae143d66b873c08883dc6382fac2e8ed45f6f3521f7e7282b6650216"
+ENVOY_SHA256 = "8c47c402fecb397b98bc65c5c340d5c4575f7936f9fa0796a8bf545988dbb5e7"
 
 http_archive(
     name = "envoy",
     sha256 = ENVOY_SHA256,
     strip_prefix = "envoy-" + ENVOY_SHA,
-    url = "https://github.com/envoyproxy/envoy/archive/" + ENVOY_SHA + ".tar.gz",
+    url = "https://github.com/istio/envoy/archive/" + ENVOY_SHA + ".tar.gz",
```
Comment (JimmyCYJ) [inline WORKSPACE]: Why do we use istio/envoy instead of envoyproxy/envoy? Are we going to actively update istio/envoy daily or weekly?

**Comment by kyessenov** [comment]
Piotr just wants to add 3rd step to updating envoy :)
1. istio/envoy
2. istio/proxy
3. istio/istio

File: WORKSPACE (hunk 1 above)
Comment (PiotrSikora) [reply WORKSPACE]: This is temporary, to make sure that we can have this merged in Istio as soon as possible.

The intention is to switch back to upstream once @quanjielin's PR is merged.

**Comment by PiotrSikora** [comment]
(switched back to `envoyproxy/envoy`, now that @quanjielin's PR is merged)

**Comment by PiotrSikora** [comment]
CircleCI passing, Prow broken pending https://github.com/istio/test-infra/pull/1167.

File: WORKSPACE (hunk 1 above)
Comment (lizan) [reply WORKSPACE]: Now it is merged so just use upstream master?

**Pull request merged**